src/main/java/creativei/helper/RequestHelper.java

```diff
@@ -0,0 +1,16 @@
+package creativei.helper;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RequestHelper {
+    private static final Logger logger=LoggerFactory.getLogger(RequestHelper.class);
+    public static Boolean nullStringValidation(String inquiryStr){
```

Comment:
Should be renamed to isEmptyRequestString(String)

Reply:
Issue Resolved in commit 5f8c87a